aksel.nav.no/website/components/sanity-modules/token-view/parts/utilities.ts

```diff
@@ -18,9 +18,8 @@ export const getGlobalReference = (
 ): { name: string; value: string | number } | null => {
   const globalRefs = Object.entries(docs)
     .filter(([key]) => key.startsWith("global-"))
-    .reduce(
-      (acc, [, value]) => [...acc, ...value],
-      [] as { name: string; value: string | number }[],
+    .flatMap(([, value]) =>
+      value.map(({ name, value: _value }) => ({ name, value: String(_value) })),
     );
```

Comment:
For some weird reason, `name` becomes `any` now 🙃

I found that one way of fixing it is to add type to `Object.entries`. I also think we can remove the nested `map()`:
```ts
  const globalRefs = Object.entries<
    { name: string; value: string | number; description?: string }[]
  >(docs)
    .filter(([key]) => key.startsWith("global-"))
    .flatMap(([, value]) => value);
```
This is a bit ugly though, so you probably want to extract the type.

It might be that it's actually cleaner to use `map()` and `flat()` separately in this case, b.c. then we don't get that type issue:
```ts
  const globalRefs = Object.entries(docs)
    .filter(([key]) => key.startsWith("global-"))
    .map(([, value]) => value)
    .flat();
```

But maybe you can find a better way to fix that type bug...

Reply:
Cound not find a better way to preserve types, but biome complains when not using flatmap, so had to "hack" it by adding the `flat()` in the return
```
const globalRefs = Object.entries(docs)
    .filter(([key]) => key.startsWith("global-"))
    .map(([, value]) => value);

  return (
    globalRefs
      .flat()
      .find(
        ({ value, name }) =>
          semanticValue === value && notBlacklistedName(name),
      ) ?? null
  );
```

Reply:
Seems like you forgot to push the changes, so I will do it.